vault/login_mfa.go

```diff
@@ -796,7 +796,7 @@ func (b *LoginMFABackend) handleMFALoginValidate(ctx context.Context, req *logic
 	}
 
 	// MFA validation has passed. Let's generate the token
-	resp, err := b.Core.LoginMFACreateToken(ctx, cachedResponseAuth.RequestPath, cachedResponseAuth.CachedAuth, req.Data)
+	resp, err := b.Core.LoginMFACreateToken(ctx, cachedResponseAuth.RequestPath, cachedResponseAuth.CachedAuth, req.Data, req.IsInlineAuth)
```

Comment:
Besides being a typo, (condition should be negated), I think this also means that login MFA would not work with inline authentication, because Login MFA returns a token for subsequent use to provide the second factor.

Reply:
Ah, that's incorrect. We can use the `X-Vault-MFA` parameter to occasionally do one-shot MFA. I've updated the code to err appropriately.